public/src/helpers/call_response_validation.jsx

```diff
@@ -0,0 +1,33 @@
+export default function validateCall(values) {
+  let errors = {};
+  const { responses, question_total, outcome } = values;
+  if (outcome !== 'ANSWERED') {
+    return errors;
+  } else if (responses) {
+    errors.responses = [];
+    for (let i = 0; i < question_total; i += 1) {
+      const question = responses[i];
+      if (typeof question === 'undefined' || typeof question.response === 'undefined') {
```

Comment:
A good check for text type questions would be a `!(question.response.strip())` to remove whitespace and then doing a falsy check (bang-bang or !!) check. This will help us avoid the case where someone avoids filling a paragraph question by putting spaces or something.

Reply:
Good point, I'll make sure to add that :)